### PR TITLE
[chrome-api-jstocxxtest] Add tests with terminated connection

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**
Add tests that calls methods using connection that was already terminated with Disconnect. Check that those will return INVALID_HANDLE. This contributes to #947.

### CODE REVIEW

**Comment by ViktoriiaKovalova** [comment]
The diff is disgusting, I'm sorry :(

**Comment by ViktoriiaKovalova** [comment]
I moved all the non-functional changes to https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/pull/1077, so this diff is readable now :)

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 1]
```diff
@@ -1021,6 +1021,104 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+            .$waitAndVerify();
+      },
+
+      // Test that GetAttrib requested on an already terminated connection
+      // returns INVALID_HANDLE error.
+      'testGetAttrib': async function() {
+        expectReportDataResult(
+            /*requestId=*/ 201, new ArrayBuffer(0),
+            /*resultCode=*/ 'INVALID_HANDLE');
+
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:1068]: nit: Some test cases have this blank line, others don't. Let's make it consistent.

**Review by emaxx-google — APPROVED** [review]
thanks

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 1 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js:1071]: done

**Pull request merged**